src/mono/wasm/runtime/jiterpreter-support.ts

```diff
@@ -482,13 +482,16 @@ export class WasmBuilder {
         return result;
     }
 
-    _generateImportSection () {
+    _generateImportSection (includeFunctionTable?: boolean) {
```

Comment:
I recommend standardizing the annotation of optional parameters -- either implicitly using the `?` symbol or explicitly using `| undefined`.